admin/views/school/pupil-register.ejs

```diff
@@ -16,7 +16,7 @@
         <div>
             <h1 tabindex="0" class="heading-xlarge">Pupil Register</h1>
             <a class="button button-secondary" href="/school/pupil/add"> Add single pupil </a>
-            <a class="button button-secondary" href=""> Add multiple pupils </a>
+            <a class="button button-secondary" href="/school/pupil/add-multiples"> Add multiple pupils </a>
```

Comment:
This is a batch interface: perhaps it could be called `add-batch-pupils`?

Reply:
yea sure it's more descriptive